app/assets/stylesheets/admin/loans/_tables.scss

```diff
@@ -19,7 +19,7 @@ table.breakeven-table-rampup {
   }
 }
 
-.editable-table {
+.editable-table, .editable-breakeven-table {
```

Comment:
Can we at least just use the one class here? I don't see a need for a separate one.